tensorflow_addons/image/augment_ops.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Image ops commonly used for augmentation."""
+
+import tensorflow as tf
+
+from tensorflow_addons.utils.types import TensorLike
+
+
+def blend(image1: TensorLike, image2: TensorLike, factor: float) -> TensorLike:
+    """Blend image1 and image2 using 'factor'.
+
+  Factor can be above 0.0.  A value of 0.0 means only image1 is used.
+  A value of 1.0 means only image2 is used.  A value between 0.0 and
+  1.0 means we linearly interpolate the pixel values between the two
+  images.  A value greater than 1.0 "extrapolates" the difference
+  between the two pixel values, and we clip the results to values
+  between 0 and 255.
+
+  Args:
+    image1: An image Tensor of type uint8.
```

Comment:
This should be enforced in the function for better UX. Could you add a check and raise a `TypeError` with an error message if the user passes the wrong type?

Reply:
I wasn't sure if we expect this to work for all types. Let's make it uint8 for now. Another thing was to checking if it's compatible with `tf.data.Dataset`. Let me know if that's necessary right now.